components/text.js

```diff
@@ -226,4 +256,11 @@ export function ZoomableImage ({ src, topLevel, ...props }) {
       {...props}
     />
   )
+
+  return (
+    (!me || !me.clickToLoadImg || isImgProxy)
```

Comment:
For UX reasons, anons don't have to click to load images and because our privacy settings are false by default. So creating an account does not suddenly require click to load for external images.